packages/connectors-web/src/alchemySmartWallet.ts

```diff
@@ -1,85 +1,233 @@
+import { alchemyTransport } from "@alchemy/common";
+import {
+  createSmartWalletClient,
+  type SmartWalletClient,
+  type SmartWalletClientEip1193Provider,
+} from "@alchemy/wallet-apis";
+import { Emitter } from "@wagmi/core/internal";
+import {
+  createWalletClient,
+  custom,
+  type Address,
+  type Chain,
+  type EIP1193Provider,
+  type ProviderConnectInfo,
+} from "viem";
 import { createConnector, type CreateConnectorFn } from "wagmi";
 
 export type AlchemySmartWalletOptions = {
-  createBaseConnector: CreateConnectorFn;
+  /** The base connector which will sign for the smart wallet. */
+  baseConnector: CreateConnectorFn;
```

Comment:
could we go ahead and add a `paymasterPolicyIds` param to the `AlchemySmartWalletOptions` now too? would be good for testing sending calls, and should be a super simple pass-through to the `smartWalletClient`.

side note: we have some type issues w/ setting call-specific policyId: https://alchemyinsights.slack.com/archives/C094T1JRHDH/p1758924853918919, but we can circle back on that later imo.